.editorconfig

```diff
@@ -16,5 +16,21 @@ fsharp_max_array_or_list_width=120
 fsharp_max_infix_operator_expression=80
 fsharp_max_value_binding_width=120
 
-[GenerateHtml.fs]
-fsharp_experimental_elmish = true
\ No newline at end of file
+[{GenerateHtml.fs,tests/FSharp.ApiDocs.Tests/files/ReferenceProject/**/*.fs}]
```

Comment:
I'd prefer a separate PR for this.

Reply:
Sure will do, just wanted to have it at least for me when working locally because I was too frustrated with dealing with indentation manually 😅